install.sh

```
@@ -1,2 +1,6 @@
 #!/bin/sh
-install -m 755 git-* /usr/local/bin
+if test -f git-paging-alias.txt;then 
+    git config --global --add include.path $(readlink -f git-paging-alias.txt)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)